evals/core/token_counting.py

```
@@ -0,0 +1,75 @@
+    The optional import stays here, in the harness analysis process. The stdlib-
+    only recording proxy never imports this module.
+    """
+    try:
+        import tiktoken
+    except ImportError:
+        return ResultTokenCount(
+            estimate_result_tokens(len(text)),
+            estimated=True,
+            method=TOKEN_ESTIMATE_METHOD,
+        )
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)